src/main/java/mate/academy/jvbookstore/validation/EmailValidator.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.jvbookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.util.regex.Pattern;
+import org.springframework.beans.factory.annotation.Value;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    @Value("${validation.email-pattern}")
+    private String pattern;
```

Comment:
The reason is not calling `compile `each time
```suggestion
    private static final String REGEX = "^(?=.{1,64}@)[A-Za-z0-9_-]+(\\.[A-Za-z0-9_-]+)*@[^-]\
        [A-Za-z0-9-]+(\\.[A-Za-z0-9-]+)*(\\.[A-Za-z]{2,})$";
    private static final Pattern PATTERN = Pattern.compile(REGEX);